syntax = "proto3";

package NCloud.NFileStore.NProtoPrivate;

option go_package = "github.com/ydb-platform/nbs/cloud/filestore/private/api/protos";

////////////////////////////////////////////////////////////////////////////////
// Actions

message TDrainNodeRequest
{
    bool KeepDown = 1;
}

message TDrainNodeResponse
{
}
